### DISCUSSION TITLE
Transaction Repeat Interval

### DISCUSSION THREAD

**Discussion by kpf61** [NickvisionApps/Denaro#757] [Q&A]
Is there anyway to set a repeat interval to 4 weeks?

**Comment by nlogozzo** [comment]
1 month is 4 weeks...

**Comment by kpf61** [comment]
Hi NickvisionApps/Denaro

Yes I know but say a transaction repeats every 28 days thats a 4 week
period, which actually gives rise to 13 periods in a year, different from
calendar year/months. How can I set that up?

[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
08/12/23,
03:29:15

**Reply by nlogozzo** [reply]
In that case you'll have to wait for #542
